geps/gep-2895/index.md

```diff
@@ -0,0 +1,250 @@
+# GEP 2895: Query Parameter Filter
+
+* Issue: [#2895](https://github.com/kubernetes-sigs/gateway-api/issues/2895)
+* Status: Experimental
+
+## TLDR
+
+Just like modify header is useful, the same goes for query parameters. 
+It's helpful to have a `HTTPQueryParamFilter` field in `HTTPRouteFilter` to set, 
+add and remove a query parameter of the HTTP request before it is sent to the upstream target.
+
+## Goals
+
+* Provide a way to modify query parameters in a `HTTPRoute`.
```

Comment:
I think it is worth mentioning here we are talking about transforming the request

Reply:
Yes, it is.